igor-web/src/main/groovy/com/netflix/spinnaker/igor/filters/CorsFilter.groovy

```diff
@@ -1,56 +0,0 @@
-/*
```

Comment:
I know this was discussed above, but are you sure that the bean that Spring is actually correctly configured?  I'm pretty sure that just having the Spring-provided CorsFilter bean exist will not be enough to set the CORS headers without adding `@CrossOrigin` annotations, and even then we'll need to be very careful that the behavior stays the same as this custom class.

Reply:
I'll double check...